src/plugins/menu/_noscript.scss

```diff
@@ -10,22 +10,28 @@
 	display: none !important;
```

Comment:
@duboisp But that placeholder selector has the opposite effect.

I want to keep it hidden (as it's always been). Using ``%noscript-display-block-important`` would forcefully display it in all noscript screen and print views, which would make a random non-functioning hamburger button appear.